xExtension-YouTube/i18n/fr/ext.php

```diff
@@ -1,11 +1,11 @@
 <?php
-return 
-  array(	
+
+return array(
 	'yt_videos' => array(
 		'height' => 'Hauteur du lecteur',
 		'width' => 'Largeur du lecteur',
 		'updates' => 'Vous pouvez trouver la dernière mise à jour de l’extension sur ',
-		'show_content' => 'Display the feeds content', // @todo translate
-        'use_nocookie' => 'Use the cookie-free domain www.youtube-nocookie.com', // @todo translate
-  ),
+		'show_content' => 'Afficher le contenu du flux',
+		'use_nocookie' => 'Use the cookie-free domain www.youtube-nocookie.com', // @todo translate
```

Comment:
```suggestion
		'use_nocookie' => 'Utiliser le domaine www.youtube-nocookie.com pour éviter les cookies',
```

Reply:
Yes, I wanted to translate, but as it was not part of this PR, so…

It's done 👍